src/components/Header/TopNav/ServiceLink.js

```diff
@@ -21,26 +21,23 @@ const StyledServiceLink = styled(Link)`
   }
 `
 
-const ServiceLink = ({ path = '/' }) => {
-  const isServicePage = getIsServicePage(path)
-  const serviceTitle = isServicePage ? path.match(servicesRegExp)[0] : null
-
-  const isSpecialityPage = path.includes('speciality')
-  const service = isSpecialityPage ? getService(path) : serviceTitle
-
-  return (
-    <Fragment>
-      {isServicePage || isSpecialityPage ? (
-        <StyledServiceLink
-          to={`/${service}`}
-          title={service}
-          color={isServicePage ? 'text' : 'white'}
-        >
-          {capitalize(service)}
-        </StyledServiceLink>
-      ) : null}
-    </Fragment>
+const getServiceFromSlug = slug =>
+  Object.keys(specialitiesMap).find(speciality =>
```

Comment:
We're already exporting an `Object.keys` of `specialitiesMap` on `navLinksHelper.js` under `servicesList`, you can use that on instead ;)

Reply:
true but I'm also using the `specialitiesMap` value within the map to reference the current key, `specialitiesMap[speciality]`. I think it's much safer to use this here rather than importing the list and the map to generate this service from slug. 

Exporting `const a  = Object.keys(b);` seems on the whole fairly unnecessary as well